ingc_geonode_theme/settings.py

```diff
@@ -87,15 +87,22 @@
 
     TEMPLATES[0]['DIRS'] = template_dirs
 
-    # Override qgis report template settings for INGC
-    LOCALIZED_QGIS_REPORT_TEMPLATE = {
-        'en': os.path.join(
-            template_dirs[0], 'geosafe', 'qgis_templates',
-            'en', 'map-report.qpt'),
-        'pt': os.path.join(
-            template_dirs[0], 'geosafe', 'qgis_templates',
-            'pt', 'map-report.qpt')
-    }
+# Override qgis report template settings for INGC
+_LOCALIZED_QGIS_REPORT_TEMPLATE = {
```

Comment:
did you remove the indentation intentionally? the `template_dirs` is not defined here if `THEME_APP_PATH` equal true

Reply:
Yes, spot on! It should have been from local root